src/components/ColorPicker.tsx

```diff
@@ -377,7 +377,7 @@ export const ColorPicker = ({
           <button
             className="color-picker-label-swatch"
             aria-label={label}
-            style={color ? { "--swatch-color": color } : undefined}
+            style={color ? { "--swatch-color": color } as CSSProperties : undefined}
```

Comment:
this shouldn't be necessary as we have types for this property

Reply:
It's showing error in my code on running, that's why I change only 1 thing for now, I will revert that then